pkg/api/frontend_logging_test.go

```diff
@@ -0,0 +1,143 @@
+package api
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+	glog "github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	log "github.com/inconshreveable/log15"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
This should be removed since we're dropping Convey.

Reply:
Thanks, I'm newb to `go` here, what is the alternative?
- [ ] don't use Convey

Reply:
ahh, standard lib, gotcha

Reply:
@domasx2 If you want, I can make the changes and push them to your branch?

Reply:
thanks a lot @aknuds1, but I'd rather do it myself and learn